applications/vessel-history/src/data/config.ts

```diff
@@ -0,0 +1,137 @@
+import { Generators } from '@globalfishingwatch/layer-composer'
+import {
+  BackgroundGeneratorConfig,
+  BasemapGeneratorConfig,
+  CartoPolygonsGeneratorConfig,
+} from '@globalfishingwatch/layer-composer/dist/generators/types'
+import { ContextLayer } from 'types'
+import { AppState } from 'types/redux.types'
+
+export const MAP_BACKGROUND_COLOR = '#163f89'
+
+export const BACKGROUND_LAYER = [
+  {
+    id: 'background',
+    tileset: 'background',
+    description: 'background',
+    type: Generators.Type.Background,
+    color: MAP_BACKGROUND_COLOR,
+  } as BackgroundGeneratorConfig,
+]
+export const DEFAULT_DATAVIEWS = [
```

Comment:
This data views format is deprecated, we don't want to use the carts generator anymore.
We could use this list: https://github.com/GlobalFishingWatch/frontend/blob/master/applications/fishing-map/src/data/dataviews.md#context

Reply:
Thanks for the advise @j8seangel . We'll definitely do it that way when we implement the final map, this is a super simplified version to have available so that we can prototype the offline access mode